capitalone/rubicon
# 59 lift integration docs

closes: #59 

---

## What

Lift the integrations up to a top level section within the documentation for better awareness

## How to Test

Build the docs locally and check em out. Let me know if additional content should be added!
